### PR TITLE
fix test

### PR DESCRIPTION
**Pull request by bblodfon**

### CODE REVIEW

**Comment by bblodfon** [comment]
@RaphaelS1 Small fix?

**Review by RaphaelS1 — APPROVED** [review]
I'm tagging you to review all my PRs from now on, good catches thanks!

File: tests/testthat/test_mlr_measures.R [hunk 1]
```diff
@@ -158,14 +158,14 @@ test_that("rcll works", {
 
   # only censored rats in test set
   p = l$predict(t, row_ids = cens_ids)
```
Comment (RaphaelS1) [inline tests/testthat/test_mlr_measures.R:160]:
```suggestion
```

File: tests/testthat/test_mlr_measures.R [hunk 2]
```diff
@@ -158,14 +158,14 @@ test_that("rcll works", {
...
-  expect_numeric(KMscore)
+  expect_numeric(p$score(m))
```
Comment (RaphaelS1) [inline tests/testthat/test_mlr_measures.R:161]:
```suggestion
  expect_numeric(l$predict(t, row_ids = cens_ids)$score(m))
```

File: tests/testthat/test_mlr_measures.R [hunk 3]
```diff
@@ -158,14 +158,14 @@ test_that("rcll works", {
...
 
   # only dead rats in test set
   p = l$predict(t, row_ids = event_ids)
```
Comment (RaphaelS1) [inline tests/testthat/test_mlr_measures.R:164]:
```suggestion
```

File: tests/testthat/test_mlr_measures.R [hunk 4]
```diff
@@ -158,14 +158,14 @@ test_that("rcll works", {
...
-  expect_numeric(KMscore)
+  expect_numeric(p$score(m))
```
Comment (RaphaelS1) [inline tests/testthat/test_mlr_measures.R:165]:
```suggestion
  expect_numeric(l$predict(t, row_ids = event_ids)$score(m))
```

**Review by RaphaelS1 — COMMENTED** [review]
We can simplify even further, what do you think?

**Comment by bblodfon** [comment]
Raphael I prefer the readability of the previous version tbh

**Comment by RaphaelS1** [comment]
In general for tests I tend to prefer less lines but happy to keep as it is here!

**Pull request merged**